packages/actions/src/Concerns/CanImportRecords.php

```diff
@@ -483,4 +494,18 @@ public function getOptions(): array
     {
         return $this->evaluate($this->options);
     }
+
+    /**
+     * @throws Exception if the disk is not an S3 disk
+     */
+    private function getDisk(): string
+    {
+        $disk = $this->evaluate($this->disk);
+
+        if (config('filesystems.disks.' . $disk . '.driver') !== 's3') {
+            throw new Exception('The disk for the storage of import data can only be customised when using an S3 compatible disk.');
```

Comment:
Strictly this isn't true though, check line 356 which can support local disks.

IMO, this should be a nullable `$disk` property, null by default, and then should fallback to `config('filament.default_filesystem_disk')` when null. Then it can also be used on line 356 and we are still respecting the default disk